src/components/LeftNavigation/LeftNavigation.tsx

```diff
@@ -78,7 +78,10 @@ export class LeftNavigation extends CommonComponent<ILeftNavigationProps, any> {
 
     private getSelectedId(options: Array<ILeftNavigationOption>) {
         const selectedOptions = options.filter(option => { return option.selected ? option.id : null; });
-        return selectedOptions[0].id;
+        
+        if (selectedOptions.length > 0) {
+            return selectedOptions[0].id;
+        }
```

Comment:
Can you explicitly return some value from function?

Reply:
I returned empty string (because id is defined as string)